packages/react/src/constants/CardPropTypes.js

```diff
@@ -654,6 +654,8 @@ export const DATE_PICKER_OPTIONS = { ICON_ONLY: 'iconOnly', FULL: 'full' };
 
 export const CardPropTypes = {
   title: PropTypes.string,
+  subtitle: PropTypes.string,
+  hasTitleWrap: PropTypes.bool,
```

Comment:
nit: perhaps it makes more sense to call this `allowTitleWrap` since it depends on additional conditions to actually wrap the title

Reply:
This goes along with the convention set by Carbon of having a capability. ie. `hasSearch, hasFiltering`. I do call out that it will be ignored if subtitle is present in the docs. I am not totally against changing it if you feel it is confusing